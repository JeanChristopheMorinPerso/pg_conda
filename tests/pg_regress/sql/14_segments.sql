BEGIN;
CREATE EXTENSION pg_conda;

SELECT conda_segments('1.2.3');
SELECT conda_segments('25!1.2.3.dev6+local-3');
ROLLBACK;
