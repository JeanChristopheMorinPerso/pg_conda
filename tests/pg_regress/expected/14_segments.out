BEGIN;
CREATE EXTENSION pg_conda;
SELECT conda_segments('1.2.3');
 conda_segments 
----------------
 {1,2,3}
(1 row)

SELECT conda_segments('25!1.2.3.dev6+local-3');
 conda_segments  
-----------------
 {1,2,3,0,dev,6}
(1 row)

ROLLBACK;
